tests/services/conftest.py

```diff
@@ -51,30 +51,43 @@ def rdm_record_service():
     return current_rdm_records_service
 
 
+def _create_record(db, uploader, record_data, community, rdm_record_service):
```

Comment:
This changes were needed because by publishing with directly on the record class (`RDMRecord`) the components were not run and some fields were ignored (e.g. access)